Vagrantfile

```diff
@@ -1,17 +1,21 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
+# Testing an ArduPilot VM:
+# sim_vehicle.py # in the starting directory should start a Copter simulation
+# xterm # X11 forwarding should work
+# sim_vehicle.py --debug --gdb
+# sim_vehicle.py --valgrind
+# cd /vagrant && ./waf configure --board=px4-v2 && ./waf build --target=bin/ardusub
+# cd /vagrant ./waf configure --board=navio2 && ./waf build --target=bin/arduplane
+# cd /vagrant ./Tools/autotest/sim_vehicle.py -v ArduPlane # should test JSBSim
+
 # Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
 VAGRANTFILE_API_VERSION = "2"
 
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
   config.vm.box = "ubuntu/zesty32"
```

Comment:
Is this line still needed?

Reply:
Seemed to break when I removed it :-)

Reply:
Interesting. But shouldn't it just be `zesty32` then? Otherwise the script doesn't run, right?